api/tests/integration/application/assessments/assessment-controller_test.js

```diff
@@ -4,8 +4,6 @@ const assessmentAuthorization = require('../../../../lib/application/preHandlers
 const moduleUnderTest = require('../../../../lib/application/assessments');
 
 describe('Integration | Application | Assessments | assessment-controller', function () {
-  // TODO: Fix this the next time the file is edited.
-  // eslint-disable-next-line mocha/no-setup-in-describe
```

Comment:
**question**
Pourquoi enlever les lint mocha/no-setup-in-describe et les remettre après ?

Reply:
Excellente question, merci ! 
On les a enlevés parce que les next-lines ont changés après le formatage de Prettier. 
L'action automatisé la plus simple, la plus rapide et la plus sûre était alors de supprimer toutes les exceptions (quelques lignes), lancer prettier pour bien reformater, lancer le script d'ajout d'exception.